mlir/utils/jenkins/Jenkinsfile

```diff
@@ -183,7 +183,7 @@ String getLabelFromCodepath(String codepath) {
         // and server nodes (gfx1030)
         label = 'mlir && ( gfx1030w || gfx1030 )'
     } else if (codepath == "vanilla"){
-        label = 'mlir && gfx906'
+        label = 'mlir'
```

Comment:
Given that both `gfx900` and `gfx906` don't exist in CI anymore, let's get rid of "vanilla" all together

Reply:
Jerry wants to keep it.

Reply:
Why so? It's tested by navi2, no?